### PR TITLE
[HOLD] Migrate portscan and vulnscan Instances to Debian Buster

### PR DESCRIPTION
**Pull request by mcdonnnj**
#  #

## 🗣 Description ##

This PR updates the Packer configurations to build the `portscan` and `vulnscan` AMIs (`nessus.json` and `nmap.json`) on Debian Buster.

This PR should only be merged along with the following PRs:
https://github.com/cisagov/cyhy-runner/pull/2
https://github.com/jsf9k/cyhy-commander/pull/17

## 💭 Motivation and Context ##

Migrating [cisagov/cyhy-runner](https://github.com/cisagov/cyhy-runner) to Python 3 requires the instances it runs on to update to using Debian Buster.

## 🧪 Testing ##

I have used these updates to create AMIs that were deployed as part of testing the linked PRs.

## ✅ Checklist ##

* [x] This PR has an informative and human-readable title.
* [x] Changes are limited to a single goal - _eschew scope creep!_
* [x] All relevant type-of-change labels have been added.
* [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
* [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
* [x] All new and existing tests pass.

### CODE REVIEW

**Review by dav3r — APPROVED** [review]
Moar Buster!  Sayonara Stretch! 🛥️

**Review by jsf9k — APPROVED** [review]
I did a stretch in Chino.

Seriously, it feels good to move away from Debian Stretch.

File: ansible/roles/nessus/files/nessus_base.py [hunk 1]
```diff
@@ -160,57 +162,50 @@ def __make_request(self, target, method, payload=None, files=None):
...
         response = self.__make_request(
             POLICY_IMPORT, "POST", payload={"file": filename}
         )
         if response.status_code == OK_STATUS:
             return response.json()
-        else:
-            raise Warning(
-                "Policy import failed; response=" "{!r}".format(response.text)
-            )
+        raise Warning("Policy import failed; response={!r}".format(response.text))
```
Comment (jsf9k) [inline ansible/roles/nessus/files/nessus_base.py]:
```suggestion
        raise Warning(f"Policy import failed; response={response.text}")
```

File: ansible/roles/nessus/files/nessus_base.py [hunk 2]
```diff
@@ -160,57 +162,50 @@ def __make_request(self, target, method, payload=None, files=None):
...
 
     def upload_file(self, files):
         response = self.__make_request(FILE_UPLOAD, "POST", files=files)
         if response.status_code == OK_STATUS:
             return response.json()
-        else:
-            raise Warning("File upload failed; response=" "{!r}".format(response.text))
+        raise Warning("File upload failed; response={!r}".format(response.text))
```
Comment (jsf9k) [inline ansible/roles/nessus/files/nessus_base.py]:
```suggestion
        raise Warning(f"File upload failed; response={response.text}")
```

**Review by dav3r — APPROVED** [review]
Still 👍  from me!

File: ansible/roles/nessus/files/nessus_base.py [hunk 3]
```diff
@@ -160,57 +162,50 @@ def __make_request(self, target, method, payload=None, files=None):
...
 
     def policy_list(self):
         response = self.__make_request(POLICY_BASE, "GET")
         if response.status_code == OK_STATUS:
             return response.json()
-        else:
-            raise Warning("Policy list failed; response=" "{!r}".format(response.text))
+        raise Warning("Policy list failed; response={!r}".format(response.text))
```
Comment (jsf9k) [inline ansible/roles/nessus/files/nessus_base.py]:
```suggestion
        raise Warning(f"Policy list failed; response={response.text}")
```

File: ansible/roles/nessus/files/nessus_base.py [hunk 4]
```diff
@@ -160,57 +162,50 @@ def __make_request(self, target, method, payload=None, files=None):
...
     def destroy_session(self):
         response = self.__make_request(LOGIN, "DELETE")
         if response.status_code == OK_STATUS:
             return response
-        else:
-            raise Warning(
-                "Session destruction failed; response=" "{!r}".format(response.text)
-            )
+        raise Warning(
+            "Session destruction failed; response={!r}".format(response.text)
```
Comment (jsf9k) [inline ansible/roles/nessus/files/nessus_base.py]:
```suggestion
            f"Session destruction failed; response={response.text}"
```

**Review by jsf9k — COMMENTED** [review]
I made a few suggestions to use f-strings instead of the less fashionable `"".format()` style.

**Pull request merged**